.changeset/popular-ladybugs-pay.md

```diff
@@ -0,0 +1,7 @@
+---
+'@backstage/core-components': patch
+'@backstage/plugin-catalog-react': patch
+---
+
+- The `UserListPicker` now only overrides the user query parameters if the kind filter is either `group` or `user`
```

Comment:
this really should be two different changesets (could have been two PRs as well, but no worries)

Reply:
@freben Ill split this into two changesets. I did ask on the issue thread when i picked this up if it should be split into two separate PR's but i received no response.

Reply:
done